packages/ripple-nuxt-tide/modules/webform/conditional-logic.js

```diff
@@ -147,19 +147,38 @@ function performTriggerCheck (rule) {
       if (typeof rule.triggerValue === 'string') {
         // value
         result = (rule.modelValue === rule.triggerValue)
+        if (rule.modelValue && fieldType != null && fieldType === 'number') {
```

Comment:
Will use `typeof` to check if number work instead of pass `fieldType` through several level of functions?

Reply:
right, I am updating this.